### PR TITLE
Feat-Playlist

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added  Playlist Features:
Create New Playlist
Add Video To Playlist
Delete Video From Playlist
Delete Playlist

### CODE REVIEW

**Review by preethasri — APPROVED** [review]
good to go

File: src/frontend/context/playlistContext.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  const [playlistsState, dispatchPlaylists] = useReducer(playListReduceFunc, {
+    playlists: [],
+  });
+  useEffect(() => {
+    token &&
+      (async () => {
+        try {
+          const { data, status } = await getPlaylistsService(token);
+          if (status === 200) {
+            console.log(data,"fromgetplaylist fun")
```
Comment (Sen-442b) [inline src/frontend/context/playlistContext.jsx]: You can remove redundant debuggers

**Review by Sen-442b — APPROVED** [review]
Folder Structure Looks Clean 
Consistent Variable Naming

File: src/frontend/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .Navbar_wrapper{
-    min-width: 100vw;
+    /* min-width: 100vw; */
```
Comment (Chakravarthi0) [inline src/frontend/components/Navbar/Navbar.css:2]: You can remove this comment, if not needed

File: src/frontend/pages/Playlist/Playlist.jsx [hunk 3]
```diff
@@ -1,18 +1,78 @@
...
+            <PlaylistCard playlist={playlist} key={playlist._id} />
+          ))}
         </div>
       </div>
+      {/* {showPlaylistModal ? (
+        <PlaylistModal
+          showPlaylistModal={showPlaylistModal}
+          setShowPlaylistModal={setShowPlaylistModal}
+        />
+      ) : null} */}
```
Comment (Chakravarthi0) [inline src/frontend/pages/Playlist/Playlist.jsx:75]: You can remove this, if not needed

**Review by Chakravarthi0 — APPROVED** [review]
There are few comments and console.log statements, which can be removed.

Apart from that, everything look good.

**Pull request merged**